workspace/apps/pidp/src/app/features/access/pages/bc-provider-application/bc-provider-application.page.html

```diff
@@ -58,23 +58,26 @@ <h3>
             <ol>
               <li>
                 Go to Microsoft account and select
-                <strong>Sign in</strong>.
+                <strong>Sign in</strong>. Please use your BCProvider login and
+                not your vendor login?
```

Comment:
Seems to be error on mockup text, this should not be a question.
```suggestion
                not your vendor login.
```